pipelines/ci-build.yml

```diff
@@ -0,0 +1,204 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+name: $(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+trigger:
+  branches:
+    include:
+      - master
+
+pool:
+  name: Hosted Windows 2019 with VS2019
+  demands:
+  - msbuild
+  - vstest
+
+variables:
+  BuildConfiguration: 'release'
+
+steps:
+- task: securedevelopmentteam.vss-secure-development-tools.build-task-credscan.CredScan@2
+  displayName: 'Run CredScan'
+  inputs:
+    debugMode: false
+
+- task: NuGetCommand@2
+  displayName: 'NuGet restore'
+  enabled: false
+
+- task: DotNetCoreCLI@2
+  displayName: 'dotnet restore'
+  inputs:
+    command: restore
+    projects: '**/*.csproj'
+
+- task: MSBuild@1
+  displayName: 'Build solution to run unit test'
+  inputs:
+    configuration: release
+    clean: true
+
+- task: VSTest@2
```

Comment:
nit: dotnet test :)

Reply:
I didn't want to refactor to keep the definition close to the original one from the UI (the file is the export of the existing pipeline except for the pack step). The main reason was that I wanted to keep the change focused on publishing the symbols, so that I can detect any divergence around that and finish all 4 packages in the FHL week.

Now that symbols are fixed, I will attempt to refactor.

Reply:
fixed now. VSTest was too verbose :)